instrumentation/kamon-spring/src/main/scala/kamon/instrumentation/spring/server/HasServerInstrumentation.scala

```diff
@@ -0,0 +1,28 @@
+package kamon.instrumentation.spring.server
+
+import kamon.Kamon
+import kamon.instrumentation.http.HttpServerInstrumentation
+
+import javax.servlet.http.HttpServletRequest
+
+trait HasServerInstrumentation {
+  def getServerInstrumentation(request: HttpServletRequest): HttpServerInstrumentation
+}
+
+object HasServerInstrumentation {
+
+  class Mixin(var serverInstrumentation: HttpServerInstrumentation) extends HasServerInstrumentation {
+    override def getServerInstrumentation(request: HttpServletRequest): HttpServerInstrumentation = {
+      if (serverInstrumentation == null) {
+        // should I just create an empty config since, you know
+        // there's no settings?
+        val config = Kamon.config().getConfig("kamon.instrumentation.spring.server")
```

Comment:
To answer the question: no. There is a comment on the `reference.conf` file about it. Just follow the same practice we have on the Akka HTTP instrumentation.